buildchain/buildchain/packaging.py

```diff
@@ -175,7 +175,21 @@ def clean() -> None:
 
 def task__download_deb_packages() -> types.TaskDict:
     """Download Debian packages locally."""
-    # TODO: Clean the repository
+    def clean() -> None:
+        """Delete cache and repositories on the ISO."""
+        coreutils.rm_rf(constants.REPO_DEB_ROOT)
```

Comment:
If you `rm -rf` that directory, you also remove all the repo at once (so the loop is useless).
I think we want something like that instead:

```python
        for repository in DEB_REPOSITORIES:
            # Repository with an explicit list of packages are created by a
            # dedicated task that will also handle their cleaning, so we skip
            # them here.
            if repository.packages:
                continue
            coreutils.rm_rf(repository.rootdir)
        constants.REPO_DEB_ROOT.rmdir()
```